src/Services/IoService.php

```diff
@@ -122,10 +122,15 @@ public function writeln($message)
      * @param  null            $default
      * @return string
      */
-    public function ask($question, $default = null)
+    public function ask($question, $default = null, $hidden = false)
     {
         $question = $question instanceof Question ? $question : new Question($question, $default);
 
+        if($hidden){
```

Comment:
A space after the `if` is missing.

Reply:
Thanks for the hint, I'll push a new commit this evening.